### PR TITLE
Use staging-friendly docserver URLs

### PR DESCRIPTION
**Pull request by ilammy**
Replace (almost) all direct links to https://docs.cossacklabs.com with domain-relative paths. That way the links will work with the staging instance of the documentation server correctly, forwarding the users to the staging instance.

Currently, at one place we have a link that is a verbatim part of a warning message. That one is left intact ([`themis-server.md`](https://github.com/cossacklabs/product-docs/blob/develop/docs/themis/debugging/themis-server.md)).

### CODE REVIEW

File: docs/themis/installation/installation-from-packages.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ which can be used to redistribute Themis to systems without MSYS2.
  3. Use NSIS installer GUI (steps 1–3 below)
     to install Themis along with all necessary dependencies.
 
-![](https://docs.cossacklabs.com/files/wiki/cossack-labs-themis-win-installer.png)
+![](/files/wiki/cossack-labs-themis-win-installer.png)
```
Comment (ilammy) [inline docs/themis/installation/installation-from-packages.md:245]: This one might be fishy.

**Comment by ilammy** [comment]
Obviously, all the links to `/simulator/interactive` will not work without proper redirects (currently there are none). And they will not work on local Hugo instances, obviously.

File: docs/themis/installation/installation-from-packages.md (hunk 1 above)
Comment (vixentael) [reply docs/themis/installation/installation-from-packages.md:245]: why fishy?

**Comment by vixentael** [comment]
> Currently, at one place we have a link that is a verbatim part of a warning message. That one is left intact

can you point out on it please?

File: docs/themis/installation/installation-from-packages.md (hunk 1 above)
Comment (ilammy) [reply docs/themis/installation/installation-from-packages.md:245]: Well, that's the only link to `/files/wiki/something` that uses `https://docs.cossacklabs.com` prefix.

Now that I've thought about it, I noticed that the file *is* hosted on `docs.cossacklabs.com`, not in this repository.

**Comment by ilammy** [comment]
> > Currently, at one place we have a link that is a verbatim part of a warning message. That one is left intact
>
> can you point out on it please?

I wish it were possible to give a permalink, but GitHub UI disagrees. It's in [`debugging/themis-server.md`](https://github.com/cossacklabs/product-docs/blob/develop/docs/themis/debugging/themis-server.md), a quote near the end of the “Secure Session” section:

> You must start session on https://docs.cossacklabs.com/interactive-simulator/setup/

File: docs/themis/installation/installation-from-packages.md (hunk 1 above)
Comment (vixentael) [reply docs/themis/installation/installation-from-packages.md:245]: > I guess we should upload it here...

strongly agree

**Comment by vixentael** [comment]
> You must start session on https://docs.cossacklabs.com/interactive-simulator/setup/

I believe that the link is very old and very wrong. Let's remove it completely and change wording to `"You must start session on .. "` (as this is error text quote, I suppose)

**Pull request merged**